test/e2e/util.go

```diff
@@ -403,14 +404,13 @@ func TearDown(namespace string) {
 		return client.DeleteAllOf(clientCtx, &operatorsv1alpha1.Subscription{}, inNamespace)
 	}).Should(Succeed(), "failed to delete test subscriptions")
 
-	Eventually(func() (remaining []operatorsv1alpha1.Subscription, err error) {
-		list := &operatorsv1alpha1.SubscriptionList{}
-		err = client.List(clientCtx, list, inNamespace)
-		if list != nil {
-			remaining = list.Items
+	Eventually(func() ([]unstructured.Unstructured, error) {
+		var subscriptiongvr = schema.GroupVersionResource{Group: "operators.coreos.com", Version: "v1alpha1", Resource: "subscriptions"}
+		list, err := dynamic.Resource(subscriptiongvr).Namespace(namespace).List(context.Background(), metav1.ListOptions{})
```

Comment:
I think this logic makes sense, moving the call from the cache to the api-server. Maybe if this works well we can expand the change to the other types in `TearDown()` as well.

Reply:
Sounds good to me.  Thanks!